apis/apiextensions/v1beta1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1beta1 contains API types that extend the Crossplane API.
+// +kubebuilder:object:generate=true
+// +groupName=apiextensions.crossplane.io
+// +versionName=v1beta1
+package v1beta1
```

Comment:
A comment on the alpha and beta packages reminding folks that their schemas must be identical may help. It hasn't been super reliable in the past, but presumably better than nothing.

Reply:
Also, with the new beta types becoming the storage version (and generally "most mature" version) I think it makes sense to update all the controller code to reconcile the v1beta1 types.

Reply:
done